i18n/generate.sh

```diff
@@ -3,14 +3,13 @@
 set -e
 
 BASE_DIR=`dirname $0`
-cd $BASE_DIR
```

Comment:
This reminded me that scripts like this one are dangerous unless you add `set -u` at the beginning. Otherwise if suddenly a variable like that gets unset and later used, you may get unexpected results.